### DISCUSSION TITLE
Axis Draw Location @ Intercept != 0

### DISCUSSION THREAD

**Discussion by besselfunct** [ManimCommunity/manim#2798] [Suggestions and Proposals]
## Description of proposed feature
Option in Axes() to specify new origin:

```
axes = Axes(
    x_range=[-5, 5, 1],
    y_range=[-2, 2, 1],
    y_axis_config={
        "x_intercept": -2
    },
    x_axis_config={
        "y_intercept": 1
    },
)
```

## How can the new feature be used?
Currently, if plotting raw data with features around the origin, the axes are in the way. Ideally, this should allow for manim animations of raw data to be more tractable.

## Additional comments
This issue came up while trying to simultaneously plot traces from .csv files using Axes() defined within a Scene. My clunky workaround will be to shift all of the .csv data manually by some constant amount.

Looking at the source, I believe there might be a way to fix things by modifying `manim/mobject/coordinate_systems.py` lines 1122-1150, but I'm not familiar enough with the codebase to know if that's correct.

**Comment by hydrobeam** [comment]
`create_axis` does seem to be the right place to look, although since it's specific to the `Axes` mobject, not `NumberLine`, the change should actually be implemented in `Axes` itself, not through `y/x_axis_config` 👍. I'll take a crack at this issue in the coming days
